README.md

```diff
@@ -64,9 +64,9 @@ launch them through:
 2. The tests for this project run on various versions of Python. We recommend
    using a Python version management tool, such as
    [pyenv](https://github.com/pyenv/pyenv), to utilize multiple versions of
-   Python. Install Pyenv: https://github.com/pyenv/pyenv#installation
-3. Install the relevant versions of Python in Pyenv: `pyenv install 3.9.1, 2.7.18, 3.5.10, 3.6.12, 3.7.9, 3.8.7, 3.10.0`
-4. Make those versions available globally: `pyenv global 3.9.1, 2.7.18, 3.5.10, 3.6.12, 3.7.9, 3.8.7, 3.10.0`
+   Python. [How to install Pyenv](https://github.com/pyenv/pyenv#installation) 
+3. Install the relevant versions of Python in Pyenv: `pyenv install 2.7.18 3.5.10 3.6.15 3.7.13 3.8.13 3.9.13 3.10.5`
+4. Make those versions available globally: `pyenv global 2.7.18 3.5.10 3.6.15 3.7.13 3.8.13 3.9.13 3.10.5`
```

Comment:
This will make Python 2.7 the default version of `python`.

We probably want:

> pyenv global 3.9.13 2.7.18 3.5.10 3.6.15 3.7.13 3.8.13 3.10.5

If we want the default to be 3.9